readme.md

```diff
@@ -387,10 +394,13 @@ b.object_id
 
 #### The Bang Symbol (`!`)
 
+The bang is simply a modifier for Ruby methods, meaning that it changes how the method operates.
```

Comment:
You don't need this first sentence. Can kick this section off with "All of the Ruby data types we have discussed so far are mutable" and from there start talking about `!`